package/libyui-qt.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Nov 12 15:02:51 UTC 2020 - Stefan Hundhammer <shundhammer@suse.com>
+
+- Honor menu bar toplevel menu enabled/disabled and visibility state
```

Comment:
this line is one char too long according to changelog styleguide - https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29 ( quote `Limit your maximum line length to 67 characters.` )

Reply:
I really don't give a damn. Whoever came up with that never took our names into account.
And 67 chars; really?!? The 1970s are over.